front/app/modules/commercial/idea_custom_fields/admin/containers/projects/edit/ideaform/IdeaCustomField.tsx

```diff
@@ -234,101 +163,90 @@ export default memo<Props>(
     }, [fieldRequired, ideaCustomField, onChange]);
 
     const handleCollapseExpand = useCallback(() => {
-      onCollapseExpand(ideaCustomField.id);
+      onCollapseExpand(id);
       // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let's not disable this rule and instead have the dependency array how it should be? Alternatively, we can remove the useCallback here as I am not sure it's adding too much value.

Reply:
Nice catch, this came from the unchanged code but it is a good improvement.